src/solver.jl

```diff
@@ -40,19 +56,25 @@ end
 Create a solver, where the summation-by-parts (SBP) operators are of order `accuracy_order` and
 associated to the `mesh`.
 """
-function Solver(mesh, accuracy_order)
+function Solver(mesh, accuracy_order; third = false)
```

Comment:
```suggestion
function Solver(mesh, accuracy_order)
```
I think it should be fine to always create the third derivative operator, such that we don't need this additional keyword argument as it is not really user friendly. See also below.